### DISCUSSION TITLE
Is HTTP/2 supported when using rustls?

### DISCUSSION THREAD

**Discussion by BenJeau** [actix/actix-web#2330] [Q&A] [answered]
Hi, I'm looking at the website and it says that only HTTP/2 is supported with `openssl`. But looking at the issues, such as https://github.com/actix/actix-web/issues/36, it seems like HTTP/2 is supported with `rustls` too. 

Am I reading the website wrong or is it true or is it simply outdated?

Talking about this line of the website https://github.com/actix/actix-website/blame/master/content/docs/http2.md#L14

**Comment by robjtede** [comment, marked as answer]
it is outdated, a PR on the website is welcome

**Reply by BenJeau** [reply]
Sounds good, thanks for the fast answer, will do a PR
